jsonlogic.go

```diff
@@ -0,0 +1,832 @@
+package jsonlogic
+
+import (
+	"errors"
+	"math"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+func less(a, b interface{}) bool {
```

Comment:
I think it will always return `false` if receives a integer.

Reply:
You rigth. It does not support integers. I did it this way because golang library for decode json always read numbers as float64. The golang spec do this conversions:

> bool, for JSON booleans
> float64, for JSON numbers
> string, for JSON strings
> []interface{}, for JSON arrays
> map[string]interface{}, for JSON objects
> nil for JSON null

Reply:
Alright. Good To go 🥇